abcclassroom/git_utils.py

```diff
@@ -12,8 +12,28 @@
 
 import github3 as gh3
 
-from .utils import _call_git
+from .utils import input_editor
 
+def _call_git(*args, directory=None):
```

Comment:
this is the rename i'm not sure about. i liked github because there was no underscore. but also it had all github functions in it. let's chat a bit more about overall module org and such!! i'm might be missing something associated with the approach here!

Reply:
I didn't change anything here except to move the function to git_utils.py. We can certainly chat more about general refactoring, but I was trying to keep it to a minimum here so that this PR didn't get huuuuuuuuuge.

Reply:
sorry @kcranston the rename is renaming the module from github.py to git_utils.py. i should have been more clear. that is the renaming that i'd love to chat about!

and i totally appreciate non ginormous PR's :) i'm super excited to test this all out when it's ready too!! yay!